src/sortStudents.ts

```diff
@@ -1,16 +1,69 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade'
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students:Student[],
+  sortBy:SortType,
+  order:SortOrder,
+):Student[] {
+  const sortedStudents = [...students];
+  let callback: (studentA:Student, studentB:Student) => number;
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      callback = (studentA, studentB):number => {
```

Comment:
I think that there is no need to define return type again
```suggestion
      callback = (studentA, studentB) => {
```

Reply:
I wanted to avoid that too, but lint wasn't cooperating 
![image](https://user-images.githubusercontent.com/78635631/230153790-e91bbf45-ade7-41e8-b1bb-9bed06d677b5.png). I guess I will disable that for entire file.